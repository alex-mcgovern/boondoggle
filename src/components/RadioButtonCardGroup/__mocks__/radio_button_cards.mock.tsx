import {
  faArrowRightArrowLeft,
  faBank,
  faShapes,
} from "@fortawesome/pro-light-svg-icons";

import { LOREM } from "../../../../mocks/LOREM.mock";
import { Icon } from "../../Icon";

export const RADIO_BUTTON_CARDS_MOCK = [
  {
    adornment: <Icon icon={faBank} size="xl" />,
    body: LOREM.text_md,
    title: "Radio item 1",
    value: "radio_item_1",
  },
  {
    adornment: <Icon icon={faArrowRightArrowLeft} size="xl" />,
    body: LOREM.text_md,
    title: "Radio item 2",
    value: "radio_item_2",
  },
  {
    adornment: <Icon icon={faShapes} size="xl" />,
    body: LOREM.text_md,
    title: "Radio item 3",
    value: "radio_item_3",
  },
];
