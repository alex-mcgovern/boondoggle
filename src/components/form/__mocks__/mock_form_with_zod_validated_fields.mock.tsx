import { z } from "zod";

import { LOREM } from "../../../../mocks/LOREM.mock";
import { FormInput } from "../../form_input";
import { FormSubmitButton } from "../../form_submit_button";

import type { FormProps } from "..";
import type { BoxProps } from "../../box";

/** ----------------------------------------------------------------------------- */

const zodSchemaEmail = z.string().email().min(2);

/** ----------------------------------------------------------------------------- */

const WRAPPER_PROPS: BoxProps = {
  marginBottom: "spacing_3",
};

/** ----------------------------------------------------------------------------- */

type MockFormWithZodValidatedFieldsProps = Pick<
  FormProps,
  "handleFormSubmission" | "handleErrors"
>;

export const mockFormWithZodValidatedFields = ({
  handleErrors = async () => {},
  handleFormSubmission = async () => {},
}: MockFormWithZodValidatedFieldsProps): FormProps => {
  return {
    children: (
      <>
        <FormInput
          errorMessage="Enter a valid email address"
          id="email"
          label={LOREM.labelEmail()}
          name="email"
          placeholder="Enter your email address"
          validate={(value: string) => {
            return zodSchemaEmail.safeParse(value).success;
          }}
          wrapperProps={WRAPPER_PROPS}
        />

        <FormSubmitButton width="100%">Submit</FormSubmitButton>
      </>
    ),
    handleErrors,
    handleFormSubmission,
    name: LOREM.name(),
  };
};
