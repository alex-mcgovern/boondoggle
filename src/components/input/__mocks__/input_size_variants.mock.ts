import type { InputProps } from "../input.comp";

export const INPUT_SIZE_VARIANTS_MOCK: InputProps[] = [
  {
    slotRight: "arrow-right",
    id: "input-large",
    label: "Large input",
    name: "Large input",
    placeholder: "Type in me",
    size: "lg",
  },
  {
    slotRight: "arrow-right",
    id: "input-sm",
    label: "Small input",
    name: "Small input",
    placeholder: "Type in me",
    size: "sm",
  },
  {
    slotRight: "arrow-right",
    id: "input-xs",
    label: "Extra-small input",
    name: "Extra-small input",
    placeholder: "Type in me",
    size: "md",
  },
];
