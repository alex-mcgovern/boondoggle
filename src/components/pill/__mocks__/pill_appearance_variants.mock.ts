import type { PillProps } from "../pill.comp";

export const BUTTON_APPEARANCE_VARIANTS_MOCK: PillProps[] = [
  // Primary pills
  {
    slotRight: "arrow-right",
    title: "Primary",
  },
  // Secondary pills
  {
    slotRight: "arrow-right",
    title: "Secondary",
  },
  // Tertiary pills
  {
    slotRight: "arrow-right",
    title: "Tertiary",
  },
];
