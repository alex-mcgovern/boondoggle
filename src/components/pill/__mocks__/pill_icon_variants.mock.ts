export const BUTTON_ICON_VARIANTS_MOCK = [
  {
    id: "pill-icon-leading",
    slotLeft: "arrow-right",
    title: "Leading icon",
  },
  {
    id: "pill-icon-trailing",
    slotRight: "arrow-right",
    title: "Trailing icon",
  },
  {
    id: "pill-no-icon",
    title: "No icon",
  },
];
