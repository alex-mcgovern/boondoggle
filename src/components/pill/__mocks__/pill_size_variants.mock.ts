import type { PillProps } from "../pill.comp";

export const BUTTON_SIZE_VARIANTS_MOCK: PillProps[] = [
  // Large pills
  {
    id: "pill-lg-primary",
    slotRight: "arrow-right",
    title: "Click me",
  },

  // Medium pills
  {
    id: "pill-sm-primary",
    slotRight: "arrow-right",
    title: "Click me",
  },

  // Small pills
  {
    id: "pill-xs-primary",
    slotRight: "arrow-right",
    title: "Click me",
  },
];
