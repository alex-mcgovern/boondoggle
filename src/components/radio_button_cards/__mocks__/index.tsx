import {
  faArrowRightArrowLeft,
  faBank,
  faShapes,
} from "@fortawesome/pro-light-svg-icons";

import { LOREM } from "../../../../mocks/LOREM.mock";
import { Icon } from "../../icon";

export const RADIO_BUTTON_CARDS_MOCK = [
  {
    adornment: <Icon icon={faBank} size="xl" />,
    body: LOREM.text_md,
    title: "Item 1",
    value: "item_1",
  },
  {
    adornment: <Icon icon={faArrowRightArrowLeft} size="xl" />,
    body: LOREM.text_md,
    title: "Item 2",
    value: "item_2",
  },
  {
    adornment: <Icon icon={faShapes} size="xl" />,
    body: LOREM.text_md,
    title: "Item 3",
    value: "item_3",
  },
];
