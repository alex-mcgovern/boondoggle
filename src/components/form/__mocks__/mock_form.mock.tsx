import { zodResolver } from "@hookform/resolvers/zod";
import * as z from "zod";

import { LOREM } from "../../../../mocks/LOREM.mock";
import { FormInput } from "../../FormInput";
import { FormRadioButtonCardGroup } from "../../FormRadioButtonCardGroup";
import { FormSelectSingle } from "../../FormSelectSingle";
import { FormSlider } from "../../FormSlider";
import { FormSubmitButton } from "../../FormSubmitButton";
import { FormTextArea } from "../../FormTextArea";
import { RADIO_BUTTON_CARDS_MOCK } from "../../RadioButtonCardGroup/__mocks__/radio_button_cards.mock";
import { mockSelectItems } from "../../Select/__mocks__/select.mock";

import type { FormProps } from "..";
import type { BoxProps } from "../../Box";

/** ----------------------------------------------------------------------------- */

const mockFormSchema = z.object({
  amount: z.number().min(1),
  // date: z.string().min(1),
  description: z.string().min(1),
  email: z.string().email().min(2),
  radio: z.string().min(1),
  select: z.string().min(1),
});

const mockFormSchemaOptional = mockFormSchema.partial();

/** ----------------------------------------------------------------------------- */

const WRAPPER_PROPS: BoxProps = {
  marginBottom: "spacing_3",
};

/** ----------------------------------------------------------------------------- */

type MockFormProps = Pick<
  FormProps,
  "handleFormSubmission" | "handleErrors"
> & {
  withDefaultValues?: boolean;
  withOptionalFields?: boolean;
};

export const mockForm = ({
  handleErrors = async () => {},
  handleFormSubmission = async () => {},
  withDefaultValues = false,
  withOptionalFields = false,
}: MockFormProps): FormProps => {
  return {
    children: (
      <>
        {/* <FormInputDate
          defaultValue={withDefaultValues ? LOREM.dateISO : undefined}
          errorMessage="Date is required"
          id="date"
          label={LOREM.labelDate()}
          name="date"
          placeholder="Select a date"
          wrapperProps={WRAPPER_PROPS}
        /> */}
        <FormInput
          defaultValue={withDefaultValues ? LOREM.email() : undefined}
          errorMessage="Enter a valid email address"
          id="email"
          label={LOREM.labelEmail()}
          name="email"
          placeholder="Enter your email address"
          wrapperProps={WRAPPER_PROPS}
        />
        <FormTextArea
          defaultValue={withDefaultValues ? LOREM.text_xxs : undefined}
          errorMessage="Enter a valid description"
          id="description"
          label={LOREM.labelDescription()}
          name="description"
          placeholder="Enter your description"
          required={false}
          wrapperProps={WRAPPER_PROPS}
        />
        <FormSelectSingle
          defaultValue={
            withDefaultValues ? mockSelectItems({})[0].value : undefined
          }
          errorMessage="Select an option"
          id="select"
          items={mockSelectItems({})}
          label={LOREM.labelDropdown()}
          name="select"
          placeholder="Select an option from the dropdown"
          wrapperProps={WRAPPER_PROPS}
        />
        <FormSlider
          defaultValue={withDefaultValues ? [50] : undefined}
          errorMessage="Pick a value from the range"
          id="amount"
          label={LOREM.labelSlider()}
          name="amount"
          placeholder="Enter an amount"
          wrapperProps={WRAPPER_PROPS}
        />
        <FormRadioButtonCardGroup
          defaultValue={
            withDefaultValues ? RADIO_BUTTON_CARDS_MOCK[0].value : undefined
          }
          errorMessage="Select an option"
          id="radio"
          items={RADIO_BUTTON_CARDS_MOCK}
          label={LOREM.labelRadioButtons()}
          name="radio"
          wrapperProps={WRAPPER_PROPS}
        />
        <FormSubmitButton width="100%">Submit</FormSubmitButton>
      </>
    ),
    handleErrors,
    handleFormSubmission,
    name: LOREM.name(),
    resolver: zodResolver(
      withOptionalFields ? mockFormSchemaOptional : mockFormSchema
    ),
  };
};
