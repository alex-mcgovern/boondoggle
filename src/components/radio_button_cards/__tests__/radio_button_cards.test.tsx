/** @jest-environment jsdom */
import { render } from "@testing-library/react";
import userEvent from "@testing-library/user-event";

import { RadioButtonCards } from "..";
import { RADIO_BUTTON_CARDS_MOCK } from "../__mocks__";

import type { RadioButtonCardsProps } from "..";

const onChangeMock = jest.fn();

const PROPS: RadioButtonCardsProps = {
  items: RADIO_BUTTON_CARDS_MOCK,
  name: "test",
  onChange: onChangeMock,
};

const renderComponent = ({ ...props }: RadioButtonCardsProps) => {
  return render(<RadioButtonCards {...props} />);
};

describe("<RadioButtonCards />", () => {
  describe("Basic smoke tests", () => {
    it("should render without throwing", () => {
      const { container } = renderComponent(PROPS);

      expect(container).not.toBeNull();
    });

    it("should match snapshot", () => {
      const { container } = renderComponent(PROPS);

      expect(container).toMatchSnapshot();
    });
  });
});

/** --------------------------------------------- */

describe("<RadioButtonCards />", () => {
  describe("On change functionality", () => {
    it("should call onChange when item is selected", async () => {
      const { getByRole } = renderComponent(PROPS);

      const firstRadioItem = getByRole("radio", {
        name: `${RADIO_BUTTON_CARDS_MOCK[0].title} ${RADIO_BUTTON_CARDS_MOCK[0].body}`,
      });

      await userEvent.click(firstRadioItem);

      expect(onChangeMock).toBeCalledWith(RADIO_BUTTON_CARDS_MOCK[0].value);
    });
  });
});
