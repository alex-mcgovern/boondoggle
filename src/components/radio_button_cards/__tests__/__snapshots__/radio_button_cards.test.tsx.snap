// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RadioButtonCards /> Basic smoke tests should match snapshot 1`] = `
<div>
  <div>
    <label
      class="radio_button_cards.styles_getGroupLabelStyles__vroqf12 get_sprinkles_alignItems_center__eiij7b2 get_sprinkles_color_text_low_contrast__eiij7b1k get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_fontSize_body_sm__eiij7b22 get_sprinkles_fontWeight_normal__eiij7b2h get_sprinkles_gap_spacing_0.5_mobile__eiij7b9x get_sprinkles_marginBottom_spacing_0.5__eiij7b45"
      for="test"
      id="label-test"
    >
      test
    </label>
    <fieldset
      aria-labelledby="label-test"
      class="get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_flexWrap_wrap__eiij7b1y get_sprinkles_gap_spacing_1_mobile__eiij7ba0"
      id="test"
    >
      <label
        class="radio_button_cards.styles_radioButtonLabelStyles__vroqf17 a11y_a11yFocus__r8he5t2 radio_button_cards.styles__vroqf16 get_sprinkles_alignItems_center__eiij7b2 get_sprinkles_aspectRatio_square__eiij7b6 get_sprinkles_border_border_default__eiij7br get_sprinkles_borderRadius_sm__eiij7b10 get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_gap_spacing_2_mobile__eiij7ba6 get_sprinkles_justifyContent_center__eiij7b39 get_sprinkles_padding_spacing_2_mobile__eiij7bc6 get_sprinkles_width_100%_mobile__eiij7bgl get_sprinkles_position_relative_mobile__eiij7bg9"
        for="test-radio_item_1"
      >
        <input
          class="radio_button_cards.styles_radioButtonInputStyles__vroqf15"
          id="test-radio_item_1"
          name="test"
          type="radio"
          value="radio_item_1"
        />
        <div
          class="get_sprinkles_alignItems_center__eiij7b2 get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_flexDirection_column_mobile__eiij7b9f get_sprinkles_justifyContent_center__eiij7b39 get_sprinkles_textAlign_center__eiij7b87"
        >
          <div
            class="get_sprinkles_marginBottom_spacing_2__eiij7b48"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-building-columns fa-xl icon.styles_icon__15uu6ts0 get_sprinkles_color_currentColor__eiij7b1o get_sprinkles_flexShrink_0__eiij7b1w"
              data-icon="building-columns"
              data-prefix="fal"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M263.9 2.1C259-.7 253-.7 248.1 2.1L15.8 133.7C6 139.3 0 149.6 0 160.8C0 178 14 192 31.2 192H256 480.8c17.2 0 31.2-14 31.2-31.2c0-11.2-6-21.6-15.8-27.1L263.9 2.1zM306.6 160c3.5-7.3 5.4-15.4 5.4-24c0-30.9-25.1-56-56-56s-56 25.1-56 56c0 8.6 1.9 16.7 5.4 24H34.3L256 34.4 477.7 160H306.6zM256 112a24 24 0 1 1 0 48 24 24 0 1 1 0-48zM64 352c-8.8 0-16 7.2-16 16s7.2 16 16 16H448c8.8 0 16-7.2 16-16s-7.2-16-16-16V224H416V352H336V224H304V352H208V224H176V352H96V224H64V352zM40 416c-8.8 0-16 7.2-16 16s7.2 16 16 16H472c8.8 0 16-7.2 16-16s-7.2-16-16-16H40zM16 480c-8.8 0-16 7.2-16 16s7.2 16 16 16H496c8.8 0 16-7.2 16-16s-7.2-16-16-16H16z"
                fill="currentColor"
              />
            </svg>
          </div>
          <div
            class="get_sprinkles_alignItems_center__eiij7b2 get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_gap_spacing_1_mobile__eiij7ba0"
          >
            <div
              class="get_sprinkles_color_text_high_contrast__eiij7b1j"
            >
              Radio item 1
            </div>
          </div>
          <div
            class="get_sprinkles_fontSize_body_xs__eiij7b24 get_sprinkles_lineHeight_body_xs__eiij7b3i get_sprinkles_color_text_low_contrast__eiij7b1k"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </div>
        </div>
      </label>
      <label
        class="radio_button_cards.styles_radioButtonLabelStyles__vroqf17 a11y_a11yFocus__r8he5t2 radio_button_cards.styles__vroqf16 get_sprinkles_alignItems_center__eiij7b2 get_sprinkles_aspectRatio_square__eiij7b6 get_sprinkles_border_border_default__eiij7br get_sprinkles_borderRadius_sm__eiij7b10 get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_gap_spacing_2_mobile__eiij7ba6 get_sprinkles_justifyContent_center__eiij7b39 get_sprinkles_padding_spacing_2_mobile__eiij7bc6 get_sprinkles_width_100%_mobile__eiij7bgl get_sprinkles_position_relative_mobile__eiij7bg9"
        for="test-radio_item_2"
      >
        <input
          class="radio_button_cards.styles_radioButtonInputStyles__vroqf15"
          id="test-radio_item_2"
          name="test"
          type="radio"
          value="radio_item_2"
        />
        <div
          class="get_sprinkles_alignItems_center__eiij7b2 get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_flexDirection_column_mobile__eiij7b9f get_sprinkles_justifyContent_center__eiij7b39 get_sprinkles_textAlign_center__eiij7b87"
        >
          <div
            class="get_sprinkles_marginBottom_spacing_2__eiij7b48"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-arrow-right-arrow-left fa-xl icon.styles_icon__15uu6ts0 get_sprinkles_color_currentColor__eiij7b1o get_sprinkles_flexShrink_0__eiij7b1w"
              data-icon="arrow-right-arrow-left"
              data-prefix="fal"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M443.3 139.3c6.2-6.2 6.2-16.4 0-22.6l-96-96c-6.2-6.2-16.4-6.2-22.6 0s-6.2 16.4 0 22.6L393.4 112 16 112c-8.8 0-16 7.2-16 16s7.2 16 16 16l377.4 0-68.7 68.7c-6.2 6.2-6.2 16.4 0 22.6s16.4 6.2 22.6 0l96-96zm-342.6 352c6.2 6.2 16.4 6.2 22.6 0s6.2-16.4 0-22.6L54.6 400 432 400c8.8 0 16-7.2 16-16s-7.2-16-16-16L54.6 368l68.7-68.7c6.2-6.2 6.2-16.4 0-22.6s-16.4-6.2-22.6 0l-96 96c-6.2 6.2-6.2 16.4 0 22.6l96 96z"
                fill="currentColor"
              />
            </svg>
          </div>
          <div
            class="get_sprinkles_alignItems_center__eiij7b2 get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_gap_spacing_1_mobile__eiij7ba0"
          >
            <div
              class="get_sprinkles_color_text_high_contrast__eiij7b1j"
            >
              Radio item 2
            </div>
          </div>
          <div
            class="get_sprinkles_fontSize_body_xs__eiij7b24 get_sprinkles_lineHeight_body_xs__eiij7b3i get_sprinkles_color_text_low_contrast__eiij7b1k"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </div>
        </div>
      </label>
      <label
        class="radio_button_cards.styles_radioButtonLabelStyles__vroqf17 a11y_a11yFocus__r8he5t2 radio_button_cards.styles__vroqf16 get_sprinkles_alignItems_center__eiij7b2 get_sprinkles_aspectRatio_square__eiij7b6 get_sprinkles_border_border_default__eiij7br get_sprinkles_borderRadius_sm__eiij7b10 get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_gap_spacing_2_mobile__eiij7ba6 get_sprinkles_justifyContent_center__eiij7b39 get_sprinkles_padding_spacing_2_mobile__eiij7bc6 get_sprinkles_width_100%_mobile__eiij7bgl get_sprinkles_position_relative_mobile__eiij7bg9"
        for="test-radio_item_3"
      >
        <input
          class="radio_button_cards.styles_radioButtonInputStyles__vroqf15"
          id="test-radio_item_3"
          name="test"
          type="radio"
          value="radio_item_3"
        />
        <div
          class="get_sprinkles_alignItems_center__eiij7b2 get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_flexDirection_column_mobile__eiij7b9f get_sprinkles_justifyContent_center__eiij7b39 get_sprinkles_textAlign_center__eiij7b87"
        >
          <div
            class="get_sprinkles_marginBottom_spacing_2__eiij7b48"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-shapes fa-xl icon.styles_icon__15uu6ts0 get_sprinkles_color_currentColor__eiij7b1o get_sprinkles_flexShrink_0__eiij7b1w"
              data-icon="shapes"
              data-prefix="fal"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M411.4 175.5c5.9 9.9 6.1 22.2 .4 32.2s-16.3 16.2-27.8 16.2H192c-11.5 0-22.2-6.2-27.8-16.2s-5.5-22.3 .4-32.2l96-160C266.3 5.9 276.8 0 288 0s21.7 5.9 27.4 15.5l96 160zM288 32L192 192H384L288 32zM472 304H328c-4.4 0-8 3.6-8 8V456c0 4.4 3.6 8 8 8H472c4.4 0 8-3.6 8-8V312c0-4.4-3.6-8-8-8zM328 272H472c22.1 0 40 17.9 40 40V456c0 22.1-17.9 40-40 40H328c-22.1 0-40-17.9-40-40V312c0-22.1 17.9-40 40-40zM224 384A96 96 0 1 0 32 384a96 96 0 1 0 192 0zM0 384a128 128 0 1 1 256 0A128 128 0 1 1 0 384z"
                fill="currentColor"
              />
            </svg>
          </div>
          <div
            class="get_sprinkles_alignItems_center__eiij7b2 get_sprinkles_display_flex_mobile__eiij7b8o get_sprinkles_gap_spacing_1_mobile__eiij7ba0"
          >
            <div
              class="get_sprinkles_color_text_high_contrast__eiij7b1j"
            >
              Radio item 3
            </div>
          </div>
          <div
            class="get_sprinkles_fontSize_body_xs__eiij7b24 get_sprinkles_lineHeight_body_xs__eiij7b3i get_sprinkles_color_text_low_contrast__eiij7b1k"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </div>
        </div>
      </label>
    </fieldset>
  </div>
</div>
`;
